run-codemods.sh

```diff
@@ -7,7 +7,7 @@ if [ "$#" -ne 3 ]; then
   echo "    run-codemods.sh PACKAGE.JSON_PATH TSCONFIG_PATH SRC_PATH"
   exit 1
 fi
-root="${0:A:h}"
+root="$(dirname $(realpath $0))"
```

Comment:
As mentioned in the main PR comment I posted, I'm not happy requiring Mac users to install realpath (via brew coreutils).

Reply:
my github emails were being sent to junk... just saw this. I'll switch to the solution you recommend, I didn't see that one in the SO post. Thank you for finding it. Should work without installing anything on *nix and mac it seems

Reply:
@tcobbs-bentley once we confirm we're going the JS route, can you resolve these comments about the now deleted shell script? I believe we'll advise people to use `node` to invoke it since the shebang won't work on windows without special unix compat layers.